releasenotes-builder/src/test/java/com/github/checkstyle/templates/TemplateProcessorTest.java

```diff
@@ -93,7 +93,7 @@ public void testGenerateOnlyNewFeature() throws Exception {
     @Test
     public void testGenerateOnlyBug() throws Exception {
         final List<String> errors = MainProcess.runPostGenerationAndPublication(
-            createNotes(null, null, createAllNotes(), null, null), createBaseCliOptions()
+            createNotes(null, null, createAllNotes(), null, null), createBaseCliOptions("1.0")
```

Comment:
Can we keep generate All tests untouched?
Validation has nothing about generation.

Reply:
Almost all, except `testGenerateOnlyMisc()` and `testGenerateOnlyBug()`. As they are right now, they violate this new flag that we are introducing. They have a 3 number version and no `new` or `breaking compatibility` labels. Either we expect an error in the test or change the release version in those.

Will change it to expect one error.

Reply:
Done.